schema_builder.py

```
@@ -0,0 +1,372 @@
+                {
+                    "question": f"Do I need reservations for date night restaurants in {city}?",
+                    "answer": f"Yes, we highly recommend making reservations for popular date night spots in {city}, especially on weekends. Many of these restaurants book up quickly."
+                }
+            ],
+            'family-friendly': [
+                {
+                    "question": f"Which {city} restaurants have kids menus?",
+                    "answer": f"All of our recommended family-friendly restaurants in {city} offer dedicated kids menus with favorites like chicken fingers, pasta, and grilled cheese."
+                },
+                {
+                    "question": f"Do these restaurants have high chairs available?",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)